src/test/resources/ide/language/python/Example.py

```diff
@@ -0,0 +1,15 @@
+"""
+    Test description of file
+
+    I <warning>has</warning> <warning>many error</warning> in this <warning>sentnce</warning>
+"""
+
+dictionary = dict()
+dictionary['kei 1'] = '<warning>this string have</warning> <warning>a</warning> error'
+dictionary["key 2"] = "<warning>the a</warning> double <warning>quoutes</warning> error"
+
+
+def area_by_shoelace(x, y):
```

Comment:
Lets additionally check for:
* strings as arguments of functions
* templated strings
Also it is better to structure tests by format of string and the way it is used. See StringLiteral.kt for an example

Reply:
Split tests, add tests for string literal passing to function